src/components/Datatable/ControlModule/ControlModule.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { isNotEmpty } from 'ramda-adjunct';
+import styled from 'styled-components';
+
+import { getColor } from '../../../utils/helpers';
+import { Col } from '../../layout';
+import { FlexContainer } from '../../FlexContainer';
+import { BatchModule } from '../BatchModule';
+import { SearchBar } from '../../forms/SearchBar';
+import { ToolsTabItem } from '../ToolsTabItem';
+import { ControlModuleProps } from './ControlModule.types';
+import { Filters } from '../../Filters';
+import { renderSuggestionFilter } from '../../forms/SearchBar/SearchSuggestionFormats';
+
+const ControlModuleContainer = styled(FlexContainer)`
+  background-color: ${getColor('graphite3H')};
+`;
+
+const FiltersContainer = styled(FlexContainer)`
+  background-color: ${getColor('graphite5H')};
+`;
+
+const ControlModule: React.FC<ControlModuleProps> = ({
+  toolsTabItems = [],
```

Comment:
Those items are strictly given by design (Show/Hide, Filter, Groups, View), we will not provide API for custom controls. We just decide by given datatable configuration which tools we should show